velox/type/tests/FilterTest.cpp

```diff
@@ -898,3 +898,21 @@ TEST(FilterTest, mergeWithBigintMultiRange) {
     }
   }
 }
+
+TEST(FilterTest, MergeMultiRange) {
```

Comment:
naming: MergeMultiRange -> mergeMultiRange

Take a look at mergeWithBigintMultiRange test above. Would it be possible to come up with something similar?

Reply:
Yep I changed my test cases.